VotingApplication/VotingApplication.Web/Controllers/LoginController.cs

```diff
@@ -18,5 +18,10 @@ public ActionResult ForgottenPassword()
         {
             return View();
         }
+
+        public ActionResult EmailNotConfirmed()
```

Comment:
The Error pages? I'm undecided about the whole structure, honestly. I think I might have been better off going with a single `/users/` toplevel controller, then `/login` and `/register` off of them, but that then might collapse a lot of them into one controller. Having the errors in one place might make sense, though.

Problem is, with our current controller routes, that would wind up introducing more controller names into the urls. However, I believe we could stop this by moving away from the default route for the MVC controllers, and use the attribute routing instead.